services/index.js

```diff
@@ -0,0 +1,38 @@
+const axios = require('axios');
+const tmdb = require('./tmdb');
+const omdb = require('./omdb');
+
+module.exports.updateConfigurations = async () => {
+  const configResponse = await tmdb.requestConfigurations();
+  return configResponse.data;
+};
+
+module.exports.getMovies = async page => {
+  const response = await tmdb.requestDiscoverMovies(page);
+  return response.data;
+};
+
+module.exports.getMoviesData = async movies => {
+  const tmdbRequests = movies.map(movie => tmdb.requestMovieDetails(movie.id));
+  const tmdbResponses = await axios.all(tmdbRequests);
+
+  let movieData = tmdbResponses.map(response => response.data);
+
+  for await (let datum of movieData) {
```

Comment:
ESLint is mad about something, I think because it doesn't understand `for await`, so just tell it to shut up

Also, `datum` is never reassigned, so it should be `const`, I think

```suggestion
// eslint-disable-next-line no-restricted-syntax
for await (const datum of movieData) {
```

Reply:
The properties inside datum are being mutated, so I want to be explicit that this object is being mutated in some way.